documentation/modules/auxiliary/admin/ldap/vmware_vcenter_vmdir_auth_bypass.md

```diff
@@ -0,0 +1,122 @@
+## Vulnerable Application
+
+### Description
+
+This module bypasses LDAP authentication in VMware vCenter Server's
+vmdir service to add an arbitrary administrator user.
+
+Version 6.7 prior to the 6.7U3f update is vulnerable. Only installations
+upgraded from a previous release line, such as 6.0 or 6.5, are affected.
+Clean installations of 6.7 are unaffected.
+
+### Setup
+
+Tested in the wild. No setup notes available at this time.
+
+## Verification Steps
+
+Follow [Setup](#setup) and [Scenarios](#scenarios).
+
+## Actions
```

Comment:
@h00die, @tperry-r7: I'm changing how I do targets or actions in module documentation. I don't think it was correct where it was after we added `Vulnerable Application`. It should really be next to `Options` and at the same level and style.

Reply:
i think thats a good place, but @tperry-r7 is the one true answer

Reply:
I'd say targets or actions are just as important as options, too. We'll be able to parse them equally well.

Reply:
And we can elaborate more than how many of the target/action descriptions read.

Reply:
#13299

Reply:
Good call @wvu-r7 . I'll lean on you for how you think the information is best organized.

Reply:
I think this was a case of my simply forgetting to move it after switching to the new standard. I shall defer to you for best practices. :)

Reply:
Thanks, everyone!